catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/catalog/CatalogApplication.java

```diff
@@ -205,16 +205,15 @@ public void init() {
             String id = req.params(":id");
             catalogService.deleteDocument(id);
             res.status(HttpStatus.SC_OK);
-            res.body(id);
-            return res;
+            return id;
```

Comment:
❓ I feel like I'm missing something. I thought the Spark Servlet returning Strings instead of actual Responses was issue that ... sparked ... this change. Wouldn't returning just a String here cause that exact error?

Reply:
No, the problem was that it was not returning Strings but Spark response objects